### PR TITLE
Logger to utils & pacer split

### PR DESCRIPTION
**Pull request by tillpp**
moved logger to utils,
created pacer to make server.java code simpler

### CODE REVIEW

**Comment by tillpp** [comment]
i dont have writing access :,( *sad noises*

File: src/server/Server.java [hunk 1]
```diff
@@ -1,48 +1,45 @@
...
 
-public class Server {
+public class Server{
 	public static final int UPDATES_PER_SEC = 20;
 	public static final int UPDATES_TIME_NS = 1_000_000_000 / UPDATES_PER_SEC;
 	public static final float UPDATES_TIME_S = UPDATES_TIME_NS / 10e9f;
-	public static boolean running;
+
+	//set the pacer
+	public static final Pacer serverPacer = new Pacer("Server") {
```
Comment (IntegratedQuantum) [inline src/server/Server.java:16]: I think using `Server extends Pacer` would be cleaner than making an anonymous class here.

File: src/server/Server.java [hunk 2]
```diff
@@ -1,48 +1,45 @@
...
-		Cubyz.world.update();
-		// TODO: Adjust for multiple players:
-		Cubyz.world.clientConnection.serverPing(Cubyz.world.getGameTime(), Cubyz.world.getBiome((int)Cubyz.player.getPosition().x, (int)Cubyz.player.getPosition().z).getRegistryID().toString());
-		// TODO: Move this to the client, or generalize this for multiplayer.
-		Cubyz.world.seek((int)Cubyz.player.getPosition().x, (int)Cubyz.player.getPosition().y, (int)Cubyz.player.getPosition().z, ClientSettings.RENDER_DISTANCE, ClientSettings.EFFECTIVE_RENDER_DISTANCE*NormalChunk.chunkSize*2);
-
-		// TODO: Send this through the proper interface and to every player:
-		ClientEntityManager.serverUpdate(Cubyz.world.getEntities());
+	public static void stop(){
+		if(serverPacer!=null)
```
Comment (IntegratedQuantum) [inline src/server/Server.java:42]: Please follow the coding convention here:
`if (serverPacer != null)`

**Review by IntegratedQuantum — CHANGES_REQUESTED** [review]

File: src/server/Server.java (hunk 1 above)
Comment (ThilaksanThev) [reply src/server/Server.java:16]: +1

**Pull request merged**